cmd/s2i/main.go

```diff
@@ -83,6 +83,13 @@ func newCmdBuild(cfg *api.Config) *cobra.Command {
 		Use:   "build <source> <image> [<tag>]",
 		Short: "Build a new image",
 		Long:  "Build a new Docker image named <tag> (if provided) from a source repository and base image.",
+		Example: `
+# Build an application Docker image from a Git repository
+$ s2i build git://github.com/openshift/ruby-hello-world openshift/ruby-22-centos7 hello-world-app
```

Comment:
Shouldn't we then update our test scripts and most importantly origin's `examples/image-streams/image-streams-centos7.json` ?